tests/post/steps/test_volume.py

```diff
@@ -15,7 +44,86 @@ def test_deploy_storage_class(host):
 def test_deploy_operator(host):
     pass
 
+@scenario('../features/volume.feature',
+          'Test volume creation (sparseLoopDevice)')
+def test_volume_creation(host):
+    pass
+
+@scenario('../features/volume.feature',
+          'Test volume deletion (sparseLoopDevice)')
+def test_volume_deletion(host):
+    pass
+
+@scenario('../features/volume.feature', 'Test PV-protection')
+def test_pv_protection(host):
+    pass
+
+@scenario('../features/volume.feature', 'Create a volume without volume type')
+def test_no_volume_type(host):
+    pass
+
+@scenario('../features/volume.feature',
+          'Create a volume with invalid volume type')
+def test_invalid_volume_type(host):
+    pass
+
+# }}}
+# Given {{{
+
+@given(parsers.parse("a Volume '{name}' exists"))
+def volume_exist(host, name, k8s_custom_client):
+    volume = _get_volume(k8s_custom_client, name)
+    if volume is None:
```

Comment:
Python tip: when 90% of the function is indented after a `if`, it could be better to do something like:

```
if volume is not None:
    return
#blablabla
```